cmd/fleet/error.go

```diff
@@ -144,6 +145,17 @@ func NewErrorResp(err error) errResp {
 		}
 	}
 
+	// Check if we have encountered a connectivity error
+	// Predicate taken from https://github.com/golang/go/blob/master/src/net/dial_test.go#L810
```

Comment:
[Nitpick]
The link is outdated. It'd be better to pin a commit, so it won't change